required/cmi5.xml

```diff
@@ -0,0 +1,19 @@
+<courseStructure xmlns="http://www.adlnet.gov/cmi5/CourseStructure.xsd">
+    <course id="">
+        <title>
+            <langstring lang="en-US">@@title</langstring>
+        </title>
+        <description>
+            <langstring lang="en-US">@@body</langstring>
+        </description>
+    </course>
+    <au id="">
```

Comment:
is this id the ActivityId? It needs to come from the json too. Not sure where it's to be defined.